pytorch_pfn_extras/training/manager.py

```diff
@@ -519,12 +520,34 @@ def state_dict(
                                  for name in self._optimizers}
         to_save['extensions'] = {name: self._extensions[name].state_dict()
                                  for name in self._extensions}
+        import pytorch_pfn_extras as ppe
+        to_save['ppe_version'] = ppe.__version__
         return to_save
 
+    def _check_snapshot_version(self, ppe_version: Optional[str]) -> None:
+        import pytorch_pfn_extras as ppe
+        must_warn = ppe_version is None or (
+                pkg_resources.parse_version(ppe_version)
+                != pkg_resources.parse_version(ppe.__version__)
+            )
```

Comment:
I'd like to avoid `import pkg_resources` as it is slow (as we got report in CuPy).
How about just comparing it as string? `if ppe_version != pytorch_pfn_extras.__version__`